<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>881</width>
     <height>496</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QFrame {
	background: rgb(186, 189, 182);
    border-style: solid;
    border-color: rgb(255, 255, 255);
    border-width: 1px;
    border-radius: 6px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="dialog_title">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>5</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Hole Operation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>435</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDoneEditing">
    <property name="geometry">
     <rect>
      <x>785</x>
      <y>435</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="GeneralParamsWidget" name="generalparamswidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>336</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>276</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Operation Parameters</string>
    </property>
    <widget class="QLabel" name="label_180">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>190</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>mm/min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="FloatLineEdit" name="z_feed_input">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>190</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>33</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="default_value" stdset="0">
      <double>100.000000000000000</double>
     </property>
     <property name="format_string" stdset="0">
      <string>.0f</string>
     </property>
     <property name="bottom" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="custom_field_unit">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>110</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_172">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>190</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Z Feed:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="FloatLineEdit" name="retract_height_input">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>150</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>33</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="default_value" stdset="0">
      <double>100.000000000000000</double>
     </property>
     <property name="format_string" stdset="0">
      <string>.0f</string>
     </property>
     <property name="bottom" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_181">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>150</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_174">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>30</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Z Start:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="FloatLineEdit" name="custom_field_input">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>110</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>33</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="default_value" stdset="0">
      <double>100.000000000000000</double>
     </property>
     <property name="format_string" stdset="0">
      <string>.0f</string>
     </property>
     <property name="bottom" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_173">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>150</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Retract:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="FloatLineEdit" name="z_start_input">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>30</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>33</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="default_value" stdset="0">
      <double>100.000000000000000</double>
     </property>
     <property name="format_string" stdset="0">
      <string>.0f</string>
     </property>
     <property name="bottom" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="custom_input_label">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>110</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Custom Field:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="FloatLineEdit" name="z_end_input">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>70</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>33</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="default_value" stdset="0">
      <double>100.000000000000000</double>
     </property>
     <property name="format_string" stdset="0">
      <string>.0f</string>
     </property>
     <property name="bottom" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_175">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>70</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Z End:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeneralParamsWidget</class>
   <extends>QWidget</extends>
   <header>widgets.conversational.general_params</header>
  </customwidget>
  <customwidget>
   <class>FloatLineEdit</class>
   <extends>VCPLineEdit</extends>
   <header>widgets.conversational.float_line_edit</header>
  </customwidget>
  <customwidget>
   <class>VCPLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qtpyvcp.widgets.input_widgets.line_edit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
